src/cml.js

```diff
@@ -163,6 +163,7 @@ class CML {
     const triggerSha = await this.triggerSha();
     const {
       commitSha: inCommitSha = triggerSha,
+      issue: issueId,
```

Comment:
bugging me as potential source of lots of future dev headache.

It's technically `issueNumber`, not `issueId`, right?

`gh issue view $NUMBER --json id --jq.id` != `$NUMBER`

Reply:
github is not the only forge we support and on bitbucket and gitlab issue id is used.

Reply:
the flag is still just `--issue` I don't feel this variable name that's not exposed to the user is a concern